Package.swift

```diff
@@ -22,10 +22,10 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(name: "MapboxDirections", url: "https://github.com/mapbox/mapbox-directions-swift.git", from: "2.2.0"),
+        .package(name: "MapboxDirections", url: "https://github.com/mapbox/mapbox-directions-swift.git", from: "2.3.0-beta.2"),
```

Comment:
`exact`? As it's exact in Cathage and SPM.

Reply:
makes sense, let's do it